### PR TITLE
Drawer dev

### PR DESCRIPTION
**Pull request by alice-0-kim**
Initial pull request

### CODE REVIEW

File: galactus.libraries.yml [hunk 1]
```diff
@@ -132,7 +132,39 @@ clf-secondary-navigation:
 clf-sticky-navigation:
   css:
     theme:
-      css/navigation.sticky.css: {}
+      css/navigation/navigation.sticky.css: {}
+  js:
+    js/navigation.sticky.js: {}
+  dependencies:
+    - core/jquery
```
Comment (occupant) [inline galactus.libraries.yml]: We can take out the jquery dependency on 138 and 139 as it's vanilla javascript

File: galactus.libraries.yml [hunk 2]
```diff
@@ -132,7 +132,39 @@ clf-secondary-navigation:
...
+      css/navigation/navigation.double.css: {}
+
+### Slide-in Navigation ###
+clf-slidein-navigation:
+  css:
+    theme:
+      css/navigation/navigation.slidein.css: {}
+  js:
+    js/navigation.slidein.js: {}
+  dependencies:
```
Comment (occupant) [inline galactus.libraries.yml]: We can remove the jquery dependency here as well.

File: galactus.theme [hunk 3]
```diff
@@ -245,6 +265,25 @@ function galactus_preprocess_page(&$variables) {
...
+  elseif (theme_get_setting('clf_navigation_placement') == 'double') {
+    $variables['navigation_placement_double'] = TRUE;
+  }
+  elseif (theme_get_setting('clf_navigation_placement') == 'higher') {
+    $variables['navigation_placement_higher'] = TRUE;
+  }
+  else {
+    $variables['navigation_placement_drawer'] = TRUE;
+  }
+//   $variables['navigation_placement_drawer--push-right'] = theme_get_setting('clf_navigation_placement') == 'drawer--push-right' ? TRUE: FALSE;
```
Comment (occupant) [inline galactus.theme]: Not quite sure where this is at - maybe we can take a look at together.

File: sass/navigation.slidein.scss [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+$drawer--border-color: #414141;
+$drawer--inner--color: #fff;
+$drawer--inner--font-size: 16px;
+
+#button--slide-in {
+  float:right;
+  color:#000;
+  border:none;
+  background-color:rgba(0, 0, 0, 0);
+  font-family: Arial,Sans-Serif;
```
Comment (occupant) [inline sass/navigation.slidein.scss]: We can remove this line

File: sass/navigation.slidein.scss [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+  padding:10px;
+  font-size:14px;
+  line-height: 1.75em!important;
+  outline: 0;
+  &:hover {
+    background-color: #002145!important;
```
Comment (occupant) [inline sass/navigation.slidein.scss]: We should probably have any colour added as a variable inline, with the variable defined at the top of the file.

File: sass/navigation.slidein.scss [hunk 6]
```diff
@@ -0,0 +1,229 @@
...
+  position: relative;
+  z-index: $pushed-content--z-index;
+  display: block;
+  overflow: hidden;
+  //transform: translate3D(0, 0, 0);
+  transition: transform $transition-speed $transition-easing;
+}
+#pushed-content.drawer--push-left.off-canvas-drawer--is-active {
+  transform: translate3D(-$drawer--width, 0, 0);
+  box-shadow: 2px 0 2px rgba(0, 0, 0, .15);
```
Comment (occupant) [inline sass/navigation.slidein.scss]: I think this isn't rendering (as it's behind the content). We might want to remove this property.

File: theme-settings.php [hunk 7]
```diff
@@ -353,7 +353,6 @@ function galactus_form_system_theme_settings_alter(&$form, FormStateInterface &$
     '#title' => t('Navigation Settings'),
     '#group' => 'clf',
   ];
-
   $form['navigation']['clf_navigation_placement'] = [
     '#type' => 'select',
     '#title' => t('Choose the type of primary navigation that should be used on this website. | NOT YET ACTIVE'),
```
Comment (occupant) [inline theme-settings.php]: We can adjust the label to tell people it's active now :)

File: templates/layout/page.html.twig [hunk 8]
```diff
@@ -72,269 +72,295 @@
...
-    <div class="row-fluid expand" id="ubc7-global-footer">
+    <!-- End of UBC Unit Identifier -->
+    <!-- UBC Unit Navigation -->
+    <nav id="ubc7-unit-menu" role="navigation" class="navbar expand clf-required">
+      <div class="navbar-inner expand">
+        <div class="container">
+
+          {% if navigation_placement_drawer %}
+            <!-- UBC Unit Drawer Menu Button -->
+            <button style="float:right;margin-right:17px;" class="hamburger visible-md visible-lg drawer-toggle--primary" id="button--slide-in">
```
Comment (occupant) [inline templates/layout/page.html.twig]: We should remove this inline style.

**Comment by alice-0-kim** [comment]
I can go over them whenever you can! Also, I can suggest that I go over your comments first & make another pull request before you have a look at them again.

**Comment by alice-0-kim** [comment]
opened a pull request pointing to develop branch.

**Pull request closed without merging**